content/applications/sales/sales/amazon_connector/features.rst

```diff
@@ -49,14 +49,21 @@ The connector is able to:
    downloading monthly fees reports, handling disputes, or issuing refunds must be managed from
    Amazon Seller Central, as usual.
 
+.. warning::
+   On February 19, 2024, in North American marketplaces, :abbr:`FBA (fulfilled by Amazon)` orders
+   created with the *Amazon Connector*, do not get the customer name anymore. This is due to the
```

Comment:
Is it possible to be more specific here?

When we say orders "do not get the customer name anymore," does this mean Amazon does not pass the customer's name over to Sales/Delivery in Odoo anymore? What specifically is impacted from the user's perspective? Are there any changes or workarounds the user should know about?